accord-core/src/main/java/accord/local/Command.java

```diff
@@ -685,25 +819,29 @@ public Ballot accepted()
 
     public static class Committed extends Accepted
     {
-        private final ImmutableSortedSet<TxnId> waitingOnCommit;
-        private final ImmutableSortedMap<Timestamp, TxnId> waitingOnApply;
+        public static final Committed EMPTY = new Committed(SaveStatus.Committed);
```

Comment:
spoke to you about this in Slack, but don't think we should have this.  This is a C* concern for memory size tracking, but as this updates the public API for accord it means that this can't be changed once a release happens (as it would be a breaking change).

I feel C* should handle this by calling the constructor, or SerializerSupport in C* code, as this is only a concern there

Reply:
I'll remove them

Reply:
Why is it a breaking change? The empty object returned is always measurable at runtime so if it changes so will the empty size. Empty size is for in memory footprint, serialization explicitly goes through the fields.